src/main/resources/view/JobListCard.fxml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<?import javafx.geometry.*?>
+<?import javafx.scene.control.*?>
+<?import javafx.scene.image.*?>
+<?import javafx.scene.layout.*?>
+
+<!--Todo: update person-card style to card or separate into 2 kinds-->
+
+<HBox alignment="CENTER_LEFT" prefWidth="475.0" spacing="10.0" styleClass="person-card" xmlns="http://javafx.com/javafx/16" xmlns:fx="http://javafx.com/fxml/1">
+   <children>
+      <Label fx:id="idx" minWidth="-Infinity" prefHeight="17.0" prefWidth="17.0" text="1" />
+      <Label fx:id="desc" layoutX="35.0" layoutY="27.0" minWidth="-Infinity" prefWidth="100.0" text="Fix washing machine" wrapText="true" />
+<!--      <FlowPane fx:id="tags" alignment="CENTER_LEFT" prefWidth="75.0" prefWrapLength="50.0" />-->
+      <Label fx:id="rate" layoutX="51.0" layoutY="27.0" prefWidth="70.0" text="\$12.06 / 1H" wrapText="true" />
+      <Label fx:id="duration" layoutX="265.0" layoutY="27.0" prefWidth="80.0" text="PT48H" wrapText="true" />
```

Comment:
should we remove this placeholder text here so we don't accidentally show garbage data to users? e.g. if some bit of our program breaks right before setting the labels to the correct values

Reply:
It helps devs know what the labels are for when they see it in SceneBuilder